.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "[r]": {
+    "editor.defaultFormatter": "positron.positron-r",
+    "editor.formatOnType": true
```

Comment:
These are actually not right. I'd encourage you to use `usethis::use_air()` in the development version of usethis instead

Reply:
i.e. you want these instead

https://github.com/tidyverse/dplyr/blob/main/.vscode/settings.json
https://github.com/tidyverse/dplyr/blob/main/.vscode/extensions.json

Reply:
You are right and I realize now that I have to rethink this. I turned Air of for igraph on purpose, because it did reformat old styling that was not supposed to be changed on a regular basis. So putting the air.toml in here but turning air off in the settings doesn't make much sense